src/gengowatcher/main.py

```
@@ -193,19 +300,19 @@ def start_web_server():
     watcher_thread.start()
 
     try:
-        cli.run()
+        app.run()
     except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)